images/openldap/contrib/init.ldif

```diff
@@ -251,3 +251,42 @@ cn: extended-group3
 owner: cn=Person3,ou=people,ou=adextended,dc=example,dc=com
 description: Person3's Group
 member: cn=fake
+
+dn: ou=incomplete,ou=groups,ou=rfc2307,dc=example,dc=com
```

Comment:
can you make the out of scope members more obviously invalid (like `member: cn=Person1,ou=out-of-scope-organization-unit,dc=example,dc=com`), or do they have to be valid DNs?